### DISCUSSION TITLE
How to integrate Webdis in Mirth Connect

### DISCUSSION THREAD

**Discussion by testlorenzospa** [nextgenhealthcare/connect#5653] [General]
Hello.

I have the goal to integrate Redis (through Webdis) in Mirth Connect.
Searching for a source/destination connector in Mirth Connect I haven't found anything useful for my requirement.
Have you an idea on how can I integrate Redis (through Webdis) in Mirth Connect?

Thanks.

**Comment by pacmano1** [comment]
Mirth supports http calls in destinations and http listeners on the source side, Webdis is a http service, what do you mean?

**Comment by testlorenzospa** [comment]
Thanks.

My goal is to execute a search (through HL7 QBP^K22 messages) on an hospital system and retrieve the result of the search (reply through HL7 RSP^K22 messages) to send to an application.
For this requirement, at first I thought to use Redis directly.
But after a searching for a source/destination connector in Mirth Connect I haven't found anything suitable for my requirement.
However, after some other research I have found Webdis and I thought to use it.
Then, I can call Webdis from Mirth Connect. Is it true?

Thanks again.

**Reply by pacmano1** [reply]
Yes.  Another options is via redis java libraries.
